blocks/init/src/Blocks/custom/tabs/tabs-editor.scss

```
@@ -0,0 +1,18 @@
+.block-tabs {
+	&__panel-item {
+		padding: 0.5em;
+		list-style: none;
+		display: inline-block;
+		cursor: pointer;
+		border-bottom: solid 0.1em global-settings(colors, primary);
+
+		&--active {
+			font-weight: bold;
+			border-bottom: solid 0.3rem global-settings(colors, primary);
```

```suggestion
			border-bottom: solid 3px global-settings(colors, primary);
```
Let's use `px` here so you don't have a "`1.8px`" border, which could look weird due to pixel rounding.